document/core/syntax/types.rst

```diff
@@ -190,7 +193,7 @@ Memory Types
 .. math::
    \begin{array}{llll}
    \production{memory type} & \memtype &::=&
-     \limits \\
+     \idxtype~\limits.\X{it} \\
```

Comment:
I think I need to say that the idxtype matches the `it` suffix on the limits here, but I'm not sure how to do that.

Reply:
By introducing a subscript on limits, as in the Overview.

Reply:
On second thought, the more adequate notation would be parameters in parentheses, like we already use for [vectors](https://webassembly.github.io/spec/core/syntax/conventions.html#vectors):
```
limits(iv) ::= {min iv, max iv?}

memtype ::= idxtype limits(type(iv))
```
I'll update the overview with this.